composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
@weierophinney I was suggesting adding "conflict" in my comment below ;-)
https://github.com/zendframework/zend-expressive/pull/450#issuecomment-279949836